docker/Dockerfile.backend

```diff
@@ -15,6 +15,12 @@ RUN apt-get update -y && apt-get upgrade -y \
     python3-matplotlib \
     unzip \
     wget
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get install -y nodejs
+
+RUN curl -L -o google-chrome.deb https://github.com/webnicer/chrome-downloads/raw/master/x64.deb/google-chrome-stable_117.0.5938.149-1_amd64.deb
+RUN apt-get install -y ./google-chrome.deb
+RUN rm google-chrome.deb
```

Comment:
frontend tests runs in a chrome headless mode, so chrome is required for that

Reply:
is that fine to install chrome in the container, not the chrome headless? also consider we are planning to run the e2e and acceptance tests from the containers only in future, so it would be required for that also

Reply:
added to future tasks. resolving here